subsys/bluetooth/services/throughput.c

```diff
@@ -50,7 +74,7 @@ static ssize_t write_callback(struct bt_conn *conn,
 
 		if ((met_data->write_len / 1024) != kb) {
 			kb = (met_data->write_len / 1024);
-			printk("=");
+			LOG_INF("=");
```

Comment:
Replacing this `printk` with `LOG_INF` will output a line for every KB transferred, so it won't look the same. The intended behavior is to print something that shows the progress of the transfer.

I see two options:
- print the amount of KB transferred instead of a symbol
- create a callback to the application, so that the application can do print/do whatever

I would go for options 2.

Reply:
A good idea. Callback added.